renv.lock

```diff
@@ -0,0 +1,2325 @@
+{
+  "R": {
+    "Version": "4.1.2",
+    "Repositories": [
+      {
+        "Name": "CRAN",
+        "URL": "https://cran.rstudio.com"
```

Comment:
I propose pointing to a stable snapshot (either from MRAN or public RSPM). This will improve stability and prevent developers from always being prompted to upgrade packages. 

See [here](https://github.com/RConsortium/submissions-pilot2/blob/main/renv.lock#L7) for an example of this approach being used in the R submissions working group.

Reply:
good idea thanks!